v2g-liberty/rootfs/root/appdaemon/apps/load_balancer/load_balancer.py

```diff
@@ -92,21 +93,21 @@ def reset_low_timer(self):
 
     def reduce_power(self, kwargs):
         median_adjustment = int(statistics.median(self.high_values))
-        current_limit = self.rate_limiter.limit
-        new_limit = max(self.min_charge_power, current_limit - median_adjustment)
+        # current_limit = self.rate_limiter.limit
```

Comment:
Why not use the current limit any more?

Reply:
The current limit is what we wish it to be.
However, the Quasar responds very slowly, so it is not necessarily there yet.
This is why I changed it to use the "active power" instead, which is (presumably) on the way to becoming the current_limit.

Using the actual active power avoids overshoots that were happening because of the slowly responding Quasar.